packages/next/types/misc.d.ts

```diff
@@ -331,6 +331,11 @@ declare module 'next/dist/compiled/comment-json' {
   export = m
 }
 
+declare module 'next/dist/compiled/web-streams-polyfill/ponyfill' {
```

Comment:
`next/dist/compiled/web-streams-polyfill` is already pointed to ponyfill in pre-compliation you don't need to add this.